dojo/tools/scout_suite/parser.py

```diff
@@ -0,0 +1,136 @@
+__author__ = 'Hasan Tayyar Besik'
+
+# Cloned form aws_scout2 scanner
+from dojo.models import Finding
+from datetime import datetime
+import json
+from django.utils.text import Truncator
+from django.utils.html import strip_tags
+
+
+class ScoutSuiteParser(object):
+    """"ScoutSuite Wiki: https://github.com/nccgroup/ScoutSuite/wiki"""
+
+    item_data = ""
+    pdepth = 0
+
+    def __init__(self, filename, test):
+        with open(filename.temporary_file_path(), "r") as fileobj:
+            raw_data = fileobj.read()
+            raw_data = raw_data.replace("scoutsuite_results =", "")
+        data = json.loads(raw_data)
+        find_date = datetime.now()
+        dupes = {}
+
+        test_description = ""
+        account_id = data["account_id"]
+        test_description = "%s  **Account:** %s\n" % (test_description, account_id)
+        last_run = data["last_run"]
+        test_description = "%s  **Ruleset:** %s\n" % (test_description, last_run["ruleset_name"])
+        test_description = "%s  **Ruleset Description:** %s\n" % (test_description, last_run["ruleset_about"])
+
+        # Summary of Services
+        test_description = "%s\n**Services** \n\n" % (test_description)
+        for service, items in list(last_run["summary"].items()):
+            test_description = "%s\n**%s** \n" % (test_description, service.upper())
+            test_description = "%s\n* **Checked Items:** %s\n" % (test_description, items["checked_items"])
+            test_description = "%s* **Flagged Items:** %s\n" % (test_description, items["flagged_items"])
+            test_description = "%s* **Max Level:** %s\n" % (test_description, items["max_level"])
+            test_description = "%s* **Resource Count:** %s\n" % (test_description, items["resources_count"])
+            test_description = "%s* **Rules Count:** %s\n\n" % (test_description, items["rules_count"])
+        test.description = test_description
```

Comment:
This part is really good. You are providing more data aligned with the source. It's good!

Reply:
Actually it is cloned from aws_scout parser as noted at the top.